### ISSUE TITLE
Tooltip in BB-explorer for long names

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/MoBi#450]
When an entry in the BB-explorer is too long to fit in the the window, a tooltip showing the complete name would be vaey handy.

**Comment by PavelBal** [comment]
E.g. here:

![image](https://github.com/user-attachments/assets/7d5fdc43-34ca-4675-8389-73435a719dff)

**Comment by rwmcintosh** [comment]
Should all explorers have this in both MoBi and PK-Sim?

**Comment by msevestre** [comment]
Well you won't know it's too long because we have the sliders now. We overwrite the default behavior (showing a tooltip) for the sake of the sliders. Not sure how to get the best of both worlds?

**Comment by rwmcintosh** [comment]
Yeah, we won't know the 'too long'. This would have to be a visible tooltip no matter the length.

**Comment by msevestre** [comment]
this behavior should be for all bb explorer IMO

**Comment by PavelBal** [comment]
If easy to implement - 12.1
